packages/zowe-explorer/__tests__/__unit__/job/ZosJobsProvider.unit.test.ts

```diff
@@ -126,6 +166,12 @@ async function createGlobalMocks() {
         value: globalMocks.mockGetSpoolFiles,
         configurable: true,
     });
+    Object.defineProperty(vscode.window, "createInputBox", {
+        value: jest.fn(() => mockInputBox),
+        configurable: true,
+    });
```

Comment:
Would you mind updating this to use `jest.spyOn` instead?

```suggestion
    const inputBoxMock = jest.spyOn(vscode.window, "createInputBox").mockReturnValue(mockInputBox);
```

`Object.defineProperty` will redefine the value across all tests, which can cause issues with any unit tests that leverage this VSCode API.